ci/vmtest/run_selftests.sh

```diff
@@ -64,7 +64,13 @@ echo "ALLOWLIST: ${ALLOWLIST}"
 
 cd ${PROJECT_NAME}/selftests/bpf
 
-test_progs
-test_progs_noalu
-test_maps
-test_verifier
+if [ $# -eq 0 ]; then
+	test_progs
+	test_progs_noalu
+	test_maps
+	test_verifier
+else
+	for test_name in "$@"; do
+		"${test_name}"
```

Comment:
did not this will work, apparently it does

Reply:
Yep, it does work.

```sh
#!/bin/sh

test_verifier() {
  echo "running test_verifier"
}

echo "Trying to run: ${1}"
"${1}"
```

```
sh /tmp/test.sh test_verifier
> Trying to run: test_verifier
> running test_verifier
```